docs/get-details/dapps/avm/teal/index.md

```diff
@@ -71,12 +71,14 @@ int 0
 loop:
 int 1
 +
+dup
 // implement loop code
 // ...
 // check upper bound
 int 10
 <=
 bnz loop
+pop
```

Comment:
```suggestion
```

I'd probably suggest leaving the `pop` off, so the program ends with a non-zero value on the stack.

Reply:
Should I add a commentary maybe? I'm concerned people will copy/paste this snippet into their code without realizing the last counter value will be left on stack after the loop exits.

Reply:
That'd be helpful. Thank you. 👍